base-theme/layouts/shortcodes/resource_link.html

```diff
@@ -2,5 +2,16 @@
 {{- $title := index .Params 1 | safeHTML -}}
 {{- $anchor_id := index .Params 2 | default ""}}
 {{- range where $.Site.Pages "Params.uid" $uuid -}}
+{{- if eq .Params.content_type "link" -}}
+    {{- partial "external-link"
```

Comment:
This is a bit of a nitpick, but can we use underscores instead of hyphens in the file names for partials and shortcodes? I realize there are some partials that use hyphens, but we should be consistent and decide on one. I believe that we should use underscores.

I think hyphens are technically okay in partials because you call them using a string with the filename, but there are places in Hugo where hyphens are problematic. For example it's technically legal to use hyphens in YAML property names in front matter, but then you can call them from template code. Hugo will throw an error if you try and do something like `{{ .Params.hyphenated-property }}`. Underscores work fine though. I don't think there are any places (at least currently) where we would access a partial or shortcode as a direct property of an object like you do with front matter params, but it's possible something like that might exist in the future. Plus, I think it's easier to just have one formatting style when it comes to these things, to maintain consistency. I realize this is an opinion though, so feel free to ignore it or raise any counterpoints.

Reply:
This makes perfect sense to me and I agree with keeping it consistent. Thank you for sharing the rationale behind choosing underscores.